msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -1001,6 +1002,23 @@ The `extensions.runtimes` array configures the sets of runtimes and actions that
 
 To use `extensions.runtimes`, see [create add-in commands](/office/dev/add-ins/develop/create-addin-commands-unified-manifest), [configure the runtime for a task pane](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-task-pane-command), and [configure the runtime for the function command](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-function-command).
 
+### extensions.contentRuntimes
+
+The `extensions.contentRuntimes` array configures pages of content that are embedded in an Excel or PowerPoint document. 
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`id`| String | 64 characters | ✔️ | Specifies the ID for runtime. Must be unique across all `runtimes` and `contentRuntimes` objects. |
+|`code`| Object | | ✔️ | Specifies the location of content for the runtime. |
+|`code.page`| String | 2048 charaters| ✔️ | The full URL of the page that contains the content that is embedded in the document. |
+|`requestedWidth`| Number | | | The desired width between 32 and 1000 pixels. If this property isn't used, the Office application determines the width.|
+|`requestedHeight`| Number | | | The desired height between 32 and 1000 pixels. If this property isn't used, the Office application determines the height.|
+|`disableSnapshot` | Boolean | | | Specifies whether a snapshot image of your content add-in is saved with the host document. Default value is `false`, so a snapshot is saved. Set to `true` to prevent a snapshot from being saved.|
+|`requirements`| Object | | | Specifies the scopes that must be supported on the Office client in order for the content to be embedded in the document. For more information, see [Specify Office Add-in requirements in the unified manifest for Microsoft 365](/office/dev/add-ins/develop/requirements-property-unified-manifest).|
+|`requirements.capabilities`| Array | | | This property (though present under `requirements` for other objects) must not be added for `contentRuntimes` object. |
+|`requirements.scopes`| Array of enums | 1 | | Identifies the scopes in which the add-in can run and defines the Microsoft 365 applications in which the extension can run. For example, `workbook` (Excel). <br>Supported values: `workbook`, `presentation` |
```

Comment:
Hi @Rick-Kirkham,
Hope you have confirmed with the feature PM about the supported scopes.

Reply:
> Hi @Rick-Kirkham, Hope you have confirmed with the feature PM about the supported scopes.

Yes. But note that the whole "requirements" property is going to be removed from the schema for the parent "contentRuntimes". This may not happen in time for public preview, so the property may still be seen in Intellisense. So, I will change this to say that they should not use the "requirements" property when the parent is "contentRuntimes".